bigbluebutton-html5/imports/ui/components/settings/submenus/application/component.jsx

```diff
@@ -191,7 +191,7 @@ class ApplicationMenu extends BaseMenu {
               >
                 {availableLocales && availableLocales.length > 0 ? (
                   <select
-                    defaultValue={this.formatLocale(this.state.settings.locale)}
+                    defaultValue={this.state.settings.locale}
```

Comment:
Why do you remove formatLocale? Does this have to do with the issue?

Reply:
When we use this.formatLocale() in the defaultValue prop, The currently activated locale does not display in the select element if the locale has a region.

e.g: English - "en" will display in the select dropdown, but if you switch to "ja_JP" when you re-open the modal, the select wont display the correct language.

Reply:
I think you accidentally fixed #5034 :laughing:  :1st_place_medal: